examples/indexer/java/SearchTransactionsNote.java

```diff
@@ -1,27 +1,44 @@
-// SearchTransactionsNote.java
-// requires java-algorand-sdk 1.4.0 or higher (see pom.xml)
 package com.algorand.javatest.indexer;
+
 import com.algorand.algosdk.v2.client.common.IndexerClient;
-import com.algorand.algosdk.v2.client.common.Client;
+import com.algorand.algosdk.v2.client.common.Response;
+import com.algorand.algosdk.v2.client.model.Transaction;
+import com.algorand.algosdk.v2.client.model.TransactionsResponse;
+
 import org.json.JSONObject;
-import com.algorand.algosdk.util.Encoder;
 
 public class SearchTransactionsNote {
-    public Client indexerInstance = null;
-
-    // utility function to connect to a node
-    private Client connectToNetwork() {
+    /**
+     * Initialize connection to indexer.
+     */
+    private static IndexerClient connectToNetwork() {
         final String INDEXER_API_ADDR = "localhost";
-        final int INDEXER_API_PORT = 8980;
-        IndexerClient indexerClient = new IndexerClient(INDEXER_API_ADDR, INDEXER_API_PORT);
-        return indexerClient;
+        final int INDEXER_API_PORT = 8981;
+        return new IndexerClient(INDEXER_API_ADDR, INDEXER_API_PORT);
     }
+
+    /**
+     * Indexer search transaction notes example.
+     */
     public static void main(String args[]) throws Exception {
-        SearchTransactionsNote ex = new SearchTransactionsNote();
-        IndexerClient indexerClientInstance = (IndexerClient) ex.connectToNetwork();
-        byte[] notePrefix = "showing prefix".getBytes();
-        String response = indexerClientInstance.searchForTransactions().notePrefix(notePrefix).execute().toString();
-        JSONObject jsonObj = new JSONObject(response.toString());
-        System.out.println("Transaction Info: " + jsonObj.toString(2)); // pretty print json
+        IndexerClient indexerClientInstance = connectToNetwork();
+        Response<TransactionsResponse> resp = indexerClientInstance.searchForTransactions()
+                .notePrefix("showing prefix".getBytes())
+                .minRound(10894697L)
```

Comment:
Is the "L" going to cause problems in this field? It seems misplaced to me. Also in the next line and a couple other places.

Reply:
@rfustino Could you double check this criteria?